README.md

```diff
@@ -1,2 +1,41 @@
+
 # dev-welcome-pr
+
 SEF Community Contributors Welcome Pull Requests
+
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/sef-global/sef-site/issues)
+
+
+# How to make your welcome PR
+
+
+## 1. Create a MarkDown file inside the folder _profiles 
+
+* Fork this repository, create a new markdown file inside the `_profiles` folder and name it with your [sef-hive](https://sef.discourse.group/) username. It should look something like this `_profiles/<YOUR-HIVE-USERNAME>.md`. 
+
+Ex-
+
+```
+_profiles/Pasindu_Rupasinghe.md
+```
+
+
+## 2. Add your profile information
+
+
+Copy the next template into your file, delete the example data and fill the information with yours.
+
+```
+---
+hiveUserName: YOUR-HIVE USERNAME 🚩 # ex- Pasindu_Rupasinghe
+welcomeThreadEntryUrl: YOUR-WELCOME THREAD ENTRY URLwelcomeThreadEntryUrl 🚩 # ex - https://sef.discourse.group/t/welcome-please-introduce-yourself/7/67?u=pasindu_rupasinghe
+quote: YOUR-SENIOR-QUOTE # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
```

Comment:
replace this with dummy data. Add instructions separately.

Reply:
Oky @jayasanka-sack. I will remake that dummy data part.

Reply:
I went through the Github graduation readme and felt your previous version looks better. Shall we revert the last commit, if you don't mind?